modules/dkan/dkan_linkchecker/dkan_linkchecker.module

```diff
@@ -0,0 +1,159 @@
+<?php
+
+/**
+ * @file
+ * Code for the DKAN Link Checker feature.
+ */
+
+include_once 'dkan_linkchecker.features.inc';
+
+/**
+ * Implements hook_preprocess_page().
+ */
+function dkan_linkchecker_preprocess_page(&$vars) {
+  if (arg(2) == 'dkan-linkchecker-report' || arg(4) == 'dkan_linkchecker_reports') {
```

Comment:
Why the check on arg(4)?

Reply:
The size of the table is so large that I'm reducing font size so it does not require an annoying amount of scrolling. The arg(4) is to include this small font size on the views UI page for the same reason.